ui/DesignSystem/Component/List.svelte

```diff
@@ -6,6 +6,8 @@
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   export let items: any[];
 
+  export let styleHoverState: boolean = true;
```

Comment:
I see where you're coming from. I also didn't like this approach.

The thing is, it doesn't make sense to extract the `<li>` styling outside of this component, because we want the list elements to look the same. So it makes sense that the styles are encapsulated in `<List>`.

What we could do is have two specialised components, instead of one generic: `<List>` and `<ClickableList>` for example.